e2e/test/scenarios/admin/i18n/content-translation/dashboard-filters.cy.spec.ts

```diff
@@ -0,0 +1,148 @@
+import { SAMPLE_DATABASE } from "e2e/support/cypress_sample_database";
+import type {
+  DashboardDetails,
+  StructuredQuestionDetails,
+} from "e2e/support/helpers";
+
+import { germanFieldNames, germanFieldValues } from "./constants";
+import {
+  interceptContentTranslationRoutes,
+  uploadTranslationDictionary,
+} from "./helpers/e2e-content-translation-helpers";
+
+const { PRODUCTS, PRODUCTS_ID } = SAMPLE_DATABASE;
+
+const { H } = cy;
+
+const questionDetails: StructuredQuestionDetails = {
+  name: "Products question",
+  query: {
+    "source-table": PRODUCTS_ID,
+  },
```

Comment:
Nit:
If you don't need ALL results from this table, consider adding the `limit` to the query. It will improve the test performance ever so slightly.